docs/content/providers/docker.md

```diff
@@ -215,6 +215,48 @@ _Optional, Default=15_
 
 Defines the polling interval (in seconds) in Swarm Mode.
 
+### `constraints`
```

Comment:
It may be easier to include one external file with all the common examples between the provider that use constraints. It could avoid some duplication. WDYT ?

Reply:
it's not possible because the comments are different for each provider.

Reply:
ok